### PR TITLE
added for-loop exercise

### PR DESCRIPTION
**Pull request by susamerz**
added an exercise to write a for loop over a string (yes I know that is not the pythonic way of doing this, better ideas for what easy thing to do with a for loop are very welcome)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,3 +83,18 @@ Eman wen
 ```
 
 [Learn how to add a property](https://docs.python.org/3/library/functions.html#property)
+
+
+## Exercise 6: Use a loop
+Add a `.spell()` method to your class that spells out its name, with dots between the
```
Comment (wmvanvliet) [inline README.md]:
```suggestion
Add a `.spell()` method to your `Gizmo` class that uses the `print()` function to spell out its name, with dots between the 
```

File: README.md [hunk 2]
```diff
@@ -83,3 +83,18 @@ Eman wen
 ```
 
 [Learn how to add a property](https://docs.python.org/3/library/functions.html#property)
+
+
+## Exercise 6: Use a loop
+Add a `.spell()` method to your class that spells out its name, with dots between the 
+letters, using a loop.
```
Comment (wmvanvliet) [inline README.md]:
```suggestion
letters. Use a `for`-loop to implement this.
```

File: gizmo.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# file for python testing test
```
Comment (wmvanvliet) [inline gizmo.py]: Solutions should not be merged into the master branch at this time.

File: README.md [hunk 4]
```diff
@@ -98,3 +98,26 @@ A.r.i.e.l
...
+```text
+" ./subjects/mock_recording_<subject_identifier>.rec"
+``` 
+where <subject_identifier> is any string. Subject identifiers will be passed to the method as a 
+list of strings.
+
+For example:
+```python
+>>> import gizmo
+>>> subject_identifiers = ["subject1","subject2"]
```
Comment (wmvanvliet) [inline README.md]:
```suggestion
>>> subject_identifiers = ["subject1", "subject2"]
```

File: .tests/test_gizmo.py [hunk 5]
```diff
@@ -43,3 +43,10 @@ def test_ex6(capsys):
     captured = capsys.readouterr()
     assert captured.out.strip() == "F.r.a.n.k.f.u.r.t.e.r"
 
+def test_ex7():
+    """Ex7: return a list of relative paths given a list of subject identifiers"""
+    import gizmo
+    subjects = ['subj1', 'subj2']
+    flist = gizmo.relative_path(subjects)
+    assert flist[0]=="./subjects/mock_recording_subj1.rec"
+    assert flist[1]=="./subjects/mock_recording_subj2.rec"
```
Comment (wmvanvliet) [inline .tests/test_gizmo.py]:
```suggestion
    assert flist[0] == "./subjects/mock_recording_subj1.rec"
    assert flist[1] == "./subjects/mock_recording_subj2.rec"
```
(let's follow PEP8 Python style guidelines)

File: gizmo.py (hunk 3 above)
Comment (susamerz) [reply gizmo.py]: Sorry, I merged my solution branch to my test dev branch (stupid idea in hindsight), and didn't manage to not commit the solution files. So, pushing this file was not intentional, sorry it happened.

**Comment by wmvanvliet** [comment]
Nice exercises @susamerz !

**Pull request merged**